catalog/dags/common/slack.py

```diff
@@ -396,15 +396,15 @@ def on_failure_callback(context: dict) -> None:
             log.info("Forgoing Slack alert due to upstream failures")
             return
         exception_message = f"""
-*Exception*: {exception}
 *Exception Type*: `{exception.__class__.__module__}.{exception.__class__.__name__}`
+*Exception*: {exception}
 """
 
     message = f"""
 *DAG*: `{dag_id}`
 *Task*: `{task_id}`
 *Logical Date*: {logical_date.strftime('%Y-%m-%dT%H:%M:%SZ')}
-*Log*: {ti.log_url}
+*Log*: <{ti.log_url}|View Logs>
 {exception_message}
```

Comment:
This is not directly related to this PR, but why is `exception_message` interpolated here and not written out in this f-string directly? I don't see it being used anywhere else.